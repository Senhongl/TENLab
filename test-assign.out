[[1
[ CPUIntType{} ]
,2
[ CPUIntType{} ]
,3
[ CPUIntType{} ]
], 1
 2
 3
[ CPUIntType{3} ]
, 4
 5
 6
[ CPUIntType{3} ]
]
