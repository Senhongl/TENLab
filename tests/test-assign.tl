a = [1,4,2,5,7,8,3,2,1,5,7,9];
a[1:10:2] = [1,3,4,5,2];
print(a);