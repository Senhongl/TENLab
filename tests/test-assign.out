 1
 1
 2
 3
 7
 4
 3
 5
 1
 2
 7
 9
[ CPUIntType{12} ]
